### PR TITLE
Enable test case for float64 with conv1d

### LINKED ISSUE #19175: 1-D convolution of tensorflow do not support float64 exactly.
In the description of tf.nn.conv1d, it said "value: A 3D Tensor. Must be of type float32 or float64". But the float64 isn't supported correctly. Some one could try the code below:

import numpy as np
import tensorflow as tf
a = np.random.randn(12, 228, 1)
w = np.random.randn(2, 1, 18)
sess = tf.Session()
c = sess.run(tf.nn.conv1d(tf.cast(a, tf.float64), tf.cast(w, tf.float64), stride=1, padding='SAME'))
sess.close()

I have test it in tensorflow 1.4 and 1.7,  they raised the error : 
Value passed to parameter 'input' has DataType float64 not in list of allowed values: float16, float32

### PR DESCRIPTION
**Pull request by yongtang**
The float64 for conv2d support has been added to tensorflow in e3468b56d323783fdfb79fa2d6c24effc58bcaa9. (Thanks brianwa84!)
Since conv1d implementation invokes conv2d, the float64 support for conv1d is supported now as well.

This fix adds the test case for float64 support of conv1d and removes the TODO.

This fix fixes #19175.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Review by rmlarsen — APPROVED** [review]
Thanks!
